script/libtest.bash

```diff
@@ -284,4 +288,27 @@ test_kubeless_function_update() {
     update_function $func
     verify_update_function $func
 }
+test_kubeless_ingress() {
+    local func=${1:?} domain=example.com act_ingress exp_ingress
+    echo_info "TEST: ingress ${func}"
+    kubeless ingress create ing-${func} --function ${func} --hostname ${func}.${domain}
+    kubeless ingress list | fgrep -w ing-${func}
```

Comment:
It would be nice if we could actually try accessing the function via the ingress rule at this point...

Reply:
agree, it has happened to me that the ingress rule is created but I am unable to access the service. Maybe you can set the domain as `$(minikube ip).nip.io` and then curl it. If it is not possible I am fine having at least the test as it is.

Reply:
@andresmgot +1~ish, although some notes:
0) deployment doesn't include an ingress controller, obv not difficult to add,
    tho didn't want to embark into that (in this PR).
1) good idea re: `<IP>.nip.io`, then because so far I've succeeded on (still)
    supporting `dind` for integration tests (I use it locally instead of minikube fwiw),
    re: getting on of the containerized nodes' IP and alike for dind.
2) ie verifying the ingress objects should be enough for `kubeless ingress ...`
    testing coverage (granted, the whole _e2e_ testing would be nicer).

Reply:
fair enough